### PR TITLE
feat: simple Pagination cant emit onChange when change page

### PR DESCRIPTION
**Pull request by Dreamcreative**
handle ant-design 

[#32659](https://github.com/ant-design/ant-design/issues/32659)

### CODE REVIEW

File: tests/jumper.test.js [hunk 1]
```diff
@@ -31,13 +31,13 @@ describe('Pagination with jumper', () => {
...
-    expect(wrapper.state().current).toBe(10);
-    expect(onChange).not.toBeCalled();
-  });
+  // it('should not call onChange when blur input', () => {
+  //   const quickJumper = wrapper.find('.rc-pagination-options-quick-jumper');
+  //   const input = quickJumper.find('input');
+  //   input.simulate('blur');
+  //   expect(wrapper.state().current).toBe(10);
+  //   expect(onChange).not.toBeCalled();
+  // });
```
Comment (afc163) [inline tests/jumper.test.js:40]: Please update it other than remove it.

Comment (Dreamcreative) [reply tests/jumper.test.js:40]: ok!!!

**Comment by afc163** [comment]
ci broken

**Comment by Dreamcreative** [comment]
@afc163 How can I stop this warning

**Comment by afc163** [comment]
Please check codecov result.

图片

**Comment by Dreamcreative** [comment]
i'm sorry, i think i had ran test in wrong way !!!

**Pull request merged**